src/Sentry.Unity.Editor/Android/DebugSymbolUpload.cs

```diff
@@ -81,7 +86,7 @@ private string _symbolUploadTaskFormat
 
             _unityProjectPath = unityProjectPath;
             _gradleProjectPath = gradleProjectPath;
-            _gradleScriptPath = Path.Combine(_gradleProjectPath, "build.gradle");
+            _gradleScriptPath = Path.Combine(_gradleProjectPath, "launcher/build.gradle");
```

Comment:
Ah, I see. Because the SDK now modifies the `launcher/build.gradle`. Why?

Reply:
The task is now added to the topmost project - launcher that depends on unityFramework target.

We are now executing the symbol upload task after the `assembleDebug` or `assembleRelease` tasks, which do not exist in the scope of the `build.gradle` file we were modifying previously.

Reply:
That depends on whether anything else is also using the locations of those files. But we can check.

Reply:
Sentry CLI is the one looking for the `.properties` so I think it's safe to keep all that in once place.